lib/modules/datasource/rubygems/index.ts

```diff
@@ -41,13 +41,11 @@ export class RubyGemsDatasource extends Datasource {
     }
 
     try {
-      const cachedVersions = await this.versionsEndpointCache.getVersions(
-        registryUrl,
-        packageName
-      );
+      const { res: versionsResult } =
```

Comment:
can we use the old variable name to have less unrelated changes?

Reply:
It still will affect other lines because of the change of underlying structure. Because of this, I think it's better make different naming as well.